src/utils.js

```diff
@@ -50,8 +51,12 @@ export const makeArray = (key, initialValue = []) => ({
     [key]: initialValue
   },
   mutations: {
-    [`push${capitalize(key)}`]: (state, item) => {
-      state[key].push(item)
+    [`push${capitalize(key)}`]: (state, items) => {
```

Comment:
```(state, ...items) {```

then remove the isArray check and use `splice(state[key].length, 0, ...items)` ?